webpack.config.dev.js

```diff
@@ -20,7 +20,7 @@ const settings = require('./tasks/helper')
 
 module.exports = {
   entry: {
-    bundle: './app/main.js' //Our starting point for our development.
+    bundle: ['babel-polyfill','./app/main.js'] //Our starting point for our development.
```

Comment:
To address a previous comment from @garrensmith we could be more granular using `core-js` and only polyfill only what's needed. I was able to launch the UI using `bundle: ['core-js/fn/array', 'core-js/fn/symbol', 'core-js/fn/promise', './app/main.js']`

Reply:
Do you have any idea how I can found which polyfill are missings?